test/clienthellotest.c

```diff
@@ -99,8 +99,7 @@ static int test_client_hello(int currtest)
          * ClientHello is already going to be quite long. To avoid getting one
          * that is too long for this test we use a restricted ciphersuite list
          */
-        if (!TEST_true(SSL_CTX_set_cipher_list(ctx, "")))
-            goto end;
+        SSL_CTX_set_cipher_list(ctx, ""); ERR_clear_error();
```

Comment:
This should be on multiple lines, and you actually want to make sure it now tests for false.
```
if (!TEST_false(SSL_CTX_set_cipher_list(ctx,"")))
    goto end;
ERR_clear_error()
```